cmd/capibmadm/cmd/powervs/port/type.go

```diff
@@ -0,0 +1,77 @@
+/*
+ Copyright 2023 The Kubernetes Authors.
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+     http://www.apache.org/licenses/LICENSE-2.0
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+*/
+
+package port
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// PSpec defines a port.
+type PSpec struct {
+	PortID      string `json:"id"`
+	Description string `json:"description"`
+	ExternalIP  string `json:"externalIP,omitempty"`
+	IPAddress   string `json:"ipAddress"`
+	MacAddress  string `json:"macAddress"`
+	Status      string `json:"status"`
+}
+
+// PList defines a list of Ports.
+type PList struct {
+	Items []PSpec `json:"items"`
+}
+
+// ToTable converts List to *metav1.Table.
+func (portList *PList) ToTable() *metav1.Table {
+	table := &metav1.Table{
+		TypeMeta: metav1.TypeMeta{
+			APIVersion: metav1.SchemeGroupVersion.String(),
+			Kind:       "Table",
+		},
+		ColumnDefinitions: []metav1.TableColumnDefinition{
+			{
+				Name: "DESCRIPTION",
+				Type: "string",
+			},
+			{
+				Name: "EXTERNALIP",
```

Comment:
One minor comment otherwise LGTM. These options can be changed as below for better readability.

```
EXTERNALIP --> EXTERNAL IP
IPADDRESS --> IP ADDRESS
MACADDRESS --> MAC ADDRESS
PORTID --> PORT ID
```

Reply:
Adressed.